salt/modules/salt_check.py

```diff
@@ -0,0 +1,449 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+'''This custom salt module makes it easy to test salt states and highstates.
+   Author: William Cannon  william period cannon at gmail dot com'''
+import os
+import os.path
+import yaml
+import salt.client
+import salt.minion
+import salt.config
+
+
+class SaltCheck(object):
+    '''
+    This class implements the salt_check
+    '''
+
+    def __init__(self):
+        self.__opts__ = salt.config.minion_config('/etc/salt/minion')
```

Comment:
The short answer is that..you don't. Right now, unit tests are designed to operate with them. If you need, them, create an opts dictionary by hand containing the keys that you need or use the `salt.config.minion_config` call that you had earlier to generate them. I recommend the first approach.